routers/api/actions/artifacts_chunks.go

```diff
@@ -64,10 +67,16 @@ type chunkFileItem struct {
 	Path       string
 }
 
-func listChunksByRunID(st storage.ObjectStorage, runID int64) (map[int64][]*chunkFileItem, error) {
+func listChunksByRunID(st storage.ObjectStorage, runID int64, basepath string) (map[int64][]*chunkFileItem, error) {
 	storageDir := fmt.Sprintf("tmp%d", runID)
 	var chunks []*chunkFileItem
 	if err := st.IterateObjects(storageDir, func(path string, obj storage.Object) error {
+		// if basepath is settings.Actions.ArtifactStorage.MinioConfig.BasePath, it should trim it
+		if basepath != "" {
+			log.Debug("listChunksByRunID, trim basepath: %s, path: %s", basepath, path)
+			basepath = strings.TrimPrefix(basepath, "/")
+			path = strings.TrimPrefix(path, basepath+"/")
+		}
```

Comment:
It's too hacky. Why not extract the "base name" of the file and parse it?

I do not see why it makes sense to put the dir into Sscanf's string `fmt.Sscanf(path, filepath.Join(storageDir ...))`

Reply:
In current design, the artifact chunk save as `tmp{run-id}/{art-id}-{start}-{end}.chunk`. Make different run chunks in different dir. 

I think I can change it to `tmp{run-id}/tmp-{run-id}-{art-id}-{start}-{end}.chunk`. The basename of file is enough to scan.